DEV-2459: Review and Improvements (#643)
cloudposse/docs: docs/layers/identity/aws-saml.mdx

    <Steps>
      1. Place this file inside the `aws-saml` component directory (`components/terraform/aws-saml/`)
      2. Commit this to version control. The filename should match the variable configured in `stacks/catalog/aws-saml.yaml`.
    </Steps>
  </Step>

  <Step>

>     ## <StepNumber/> Deploy `aws-saml`

Reviewer: ```suggestion
    ## <StepNumber/> Deploy the SAML Integration
```

Author: added

    ```bash
    atmos terraform apply aws-saml -s core-gbl-identity
    ```
  </Step>

  <Step>

>     ## <StepNumber/> (Optional) Download AWS Extend Switch Roles.

Reviewer: ```suggestion
    ## <StepNumber/> (Optional) Download AWS Extend Switch Roles Browser Extension.
```

Author: added

>     We use the AWS Extend Switch Roles plugin to make it easier to switch roles in the AWS Console. This is optional, but recommended especially if you are not using AWS IAM Identity Center.

Reviewer: ```suggestion
    We suggest using the AWS Extend Switch Roles browser extension to simplify role-switching in the AWS Console. This is optional but particularly helpful if you’re not using AWS IAM Identity Center.
```
